src/components/survey/tally/post-purchase.tsx

```diff
@@ -0,0 +1,17 @@
+const PostPurchase = ({email}: {email: string}) => {
+  return (
+    <div className="p-5 mt-4 text-gray-800 rounded-lg dark:bg-gray-800 dark:text-gray-200">
+      <iframe
+        src={`https://survey.zacjones.dev/post-purchase?alignLeft=1&hideTitle=1&transparentBackground=1&embed=1&email=${email}&form_id=post-purchase&question=What brings you here today?`}
+        width="100%"
+        height="250px"
+        frameBorder={1}
+        marginHeight={1}
+        marginWidth={1}
+        title="Post Purchase Question"
+      ></iframe>
+    </div>
+  )
+}
+
+export default PostPurchase
```

Comment:
I have the form hosted on my domain from my tally account but can switch this over. On the tally side of things I'm overriding styles
```
<style>
h3 {
  color: #ffffff !important;
}
button {
  background-color: #2f6af4 !important;
}
</style>
```